pkg/reconciler/revision/resources/queue.go

```diff
@@ -78,6 +78,12 @@ var (
 		// thus don't want to be limited by K8s granularity here.
 		TimeoutSeconds: 10,
 	}
+
+	queueAllowPrivilegeEscalation = false
+
+	queueSecurityContext = &corev1.SecurityContext{
+		AllowPrivilegeEscalation: &queueAllowPrivilegeEscalation,
```

Comment:
```suggestion
		AllowPrivilegeEscalation: ptr.Bool(false),
```

One less thing to name 🙂

Reply:
Done